README.md

```diff
@@ -1 +1,23 @@
-# ToDoList-team6
\ No newline at end of file
+# ToDoList-team6
+
+# Our Team
+
+- [Mohammed](https://github.com/mo7amedehab97)
+- [Mosab](https://github.com/Mosab-Ahmed)
+- [Safa](https://github.com/safamohanna95)
+
+# Project description
+
+You can use To Do LIST to make shopping lists or task lists, take notes, plan an event, or set reminders to increase your productivity and focus on what matters to you, To Do LIST makes it easy to stay organized and manage your life.
+
+# User stories (purpose)
+
+- Add an infinite number of lists, list items, or notes.
```

Comment:
when you write a user story start with As a user, .....,
Example: As a user, I can add an infinite number of lists, list items, or notes.